package.json

```diff
@@ -66,7 +66,8 @@
     "manage-roles:test": "npx hardhat manage-roles --network test --env test >> logs/test.manage.roles.txt",
     "natspec-interface-id": "npx hardhat compile && node scripts/util/natspec-interfaceid.js",
     "natspec-interface-id:fix": "npx hardhat compile && node scripts/util/natspec-interfaceid.js --fix",
-    "estimate-limits": "npx hardhat clean && npx hardhat compile && npx hardhat estimate-limits"
+    "estimate-limits": "npx hardhat clean && npx hardhat compile && npx hardhat estimate-limits",
+    "prepare": "husky install"
```

Comment:
I realize the husky docs say make a one-one time script called `prepare` to get going.

However, in the context of our repo "prepare" doesn't give much context. And if you don't know what husky is for, most likely you will not run this. Because there is no documentation explaining what it is for and when / why you should run it.

Since it is to be run once at developer setup time and it needs documentation, I suggest removing the script from this file entirely and instead adding a section called "**Prepare pre-commit hooks**" in [local-development.md](https://github.com/bosonprotocol/boson-protocol-contracts/blob/main/docs/local-development.md) between the [Install required Node modules](https://github.com/bosonprotocol/boson-protocol-contracts/blob/main/docs/local-development.md#install-required-node-modules) and [Configure Environment](https://github.com/bosonprotocol/boson-protocol-contracts/blob/main/docs/local-development.md#configure-environment) sections. It should just explain what pre-commit hooks are defined and that they are installed with `husky install`

Reply:
@cliffhall Since NPM v4,`prepare` script is executed when you run npm install

> Runs BEFORE the package is packed, i.e. during npm publish and npm pack
> Runs on local npm install without any arguments
> Runs AFTER prepublish, but BEFORE prepublishOnly

Check https://docs.npmjs.com/cli/v9/using-npm/scripts

Reply:
Right. We don't need it to run this every time we run npm install though. Only the first time.

Reply:
changed and documentation added